docs/getting-started-wsl.md

```diff
@@ -0,0 +1,48 @@
+# Software Setup on Windows Subsystem for Linux
+
+## Table of Contents
+
+* [Table Of Contents](#table-of-contents)
+* [Introduction](#introduction)
+* [WSL2 Setup](#wsl2-setup)
+* [X Server Setup](#wsl2-setup)
+
+
+## Introduction
+
+Windows has a new Windows Subsystem for Linux component that can be used to develop and run code for Linux on Windows. WSL1 was a Windows component that implemented Linux kernel interfaces, and didn't work great with Thunderbots code. WSL2 runs a full-fledged Linux kernel in a VM, and works great with Thunderbots and grSim with the exception that we need to use software rendering instead of GPU-accelerated rendering for grSim.
+
+## WSL2 Setup
+
+1. You'll need to be on an Insider build 19041.207 to use WSL2. Navigate to Settings > Windows Update > Windows Insider Program and complete the steps to enrol in the Windows Insider Program if you have not already. 
```

Comment:
nit: specify 19041.207 **or later** (is there a timeline for this getting to the mainline branch)?

Reply:
A WSL2  capable build is the newest feature update on the mainline release, need to update this

Reply:
yea WSL2 is available on Windows 10 version 2004 https://docs.microsoft.com/en-us/windows/wsl/wsl2-index

Reply:
updated